### ISSUE TITLE
Model loss goes up from 0.2 to 15 after traning for 72 hours

### ISSUE THREAD

**Issue by kkggken** [iperov/DeepFaceLab#235]
![QQ浏览器截图20190414105707](https://user-images.githubusercontent.com/49333401/56086959-e0c27a80-5ea4-11e9-9916-61c140158bfa.png)
![QQ浏览器截图20190414110535](https://user-images.githubusercontent.com/49333401/56086979-41ea4e00-5ea5-11e9-98e8-3b2a90a172cf.png)

As shown in the picture, the loss goes up suddenly and the preview become blank angain.

What cause this problem? How do I avoid it in the future?

**Comment by oracle9i88** [comment]
+1

**Comment by iperov** [comment]
no solution

**Comment by YouYouExcellent** [comment]
@kkggken can you share your extract PICs ?

**Comment by visonpon** [comment]
I also encountered this problem,  but you can save the model based on epoch or iter so that you can reuse the one that work fine and then use it  to test or finetune based on it.

**Comment by kkggken** [comment]
> I also encountered this problem,  but you can save the model based on epoch or iter so that you can reuse the one that work fine and then use it  to test or finetune based on it.


@visonpon This is what I did. Luckily I save the model every 30 mins to a different folder. But the problem is no matter how I adjust the setting of the model ( tried reduce the bath size, turn off pixel loss, etc), it still collapse at some point. Some time it could train for 4 hours without problem but it could collapsed in 30 mins as well. And I found that there are actually many people have same experience not just with the SAE model but also the H128 model.

Hope iperov will adjust and fix the problem in the future.
